pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_delete-modules.adoc

```diff
@@ -0,0 +1,18 @@
+[id="delete-modules_{context}"]
+
+= Deleteing a module in {ProductShortName}
```

Comment:
s / "Deleteing" / "Deleting"

Also, I wonder how strict is the guidance for filename, id and heading to be similar in Pv2. If it is strict, the id should be changed to id="deleting-a-module-in-pantheon_{context}" and the file name proc_deleting-a-module-in-pantheon.adoc. @adahms is this something we should take care of now, or should we do this once the Pv2 rules are finalized?